lib/ansible/plugins/connection/ssh.py

```diff
@@ -297,6 +297,11 @@
     display = Display()
 
 
+NOT_SSH_ERRORS = ('Traceback (most recent call last):',  # Python-2.6 when there's an exception
+                                                         # while invoking a script via -m
+                  'PHP Parse error'  # Php always returns error 255
```

Comment:
Could you please have a trailing comma here for the sake of better future diffs?

Reply:
Yep, will do

Reply:
Will do